src/sql/parts/modelComponents/formContainer.component.ts

```diff
@@ -37,12 +41,15 @@ class FormItem {
 
 @Component({
 	template: `
-		<div #container *ngIf="items" class="form-table" [style.width]="getFormWidth()" [style.height]="getFormHeight()">
+		<div #container *ngIf="items" class="form-table" [style.padding]="getFormPadding()" [style.width]="getFormWidth()" [style.height]="getFormHeight()">
 			<ng-container *ngFor="let item of items">
 			<div class="form-row" *ngIf="isFormComponent(item)" [style.height]="getRowHeight(item)">
 
 					<ng-container *ngIf="isHorizontal(item)">
-						<div class="form-cell">{{getItemTitle(item)}}</div>
+						<div class="form-cell" [style.font-size]="getItemTitleFontSize(item)">
+							{{getItemTitle(item)}}<span class="form-required" *ngIf="isItemRequired(item)">*</span>
+							<span class="icon info" *ngIf="itemHasInfo(item)" [title]="getItemInfo(item)"></span>
```

Comment:
for  now using existing "info" icon. will change that to the right icon when it's ready

Reply:
That should be fine but how big is the icon? When I was using it for other stuff it was 20x20 pixels by default which is probably a bit big for form items

Reply:
added a screenshot. it looks fine in the form

---
In reply to: [197294014](https://github.com/Microsoft/sqlopsstudio/pull/1707#discussion_r197294014) [](ancestors = 197294014)